### PR TITLE
add future warning to dask.distributed

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

**Comment by mrocklin** [comment]
I'm pretty excited about removing this module from the codebase.

File: dask/distributed/__init__.py [hunk 1]
```diff
@@ -2,3 +2,10 @@
 from .scheduler import Scheduler
 from .client import Client
 from .ipython_utils import dask_client_from_ipclient
+
+from warnings import warn
+
+warn(FutureWarning('''
```
Comment (jcrist) [inline dask/distributed/__init__.py]: Should probably be a `DeprecationWarning` instead. `FutureWarning` is about future semantic changes, not deprecated modules.

Comment (mrocklin) [inline dask/distributed/__init__.py]: That makes sense.  Apparently DeprecationWarnings are silenced by default.
Not sure if that changes what's best.

Comment (jcrist) [inline dask/distributed/__init__.py]: Yeah, that's tricky. For some data points, SymPy subclasses DeprecationWarning, and turns on visibility for those. Bokeh raises a `UserWarning` when someone uses the deprecated `_legacy_charts`. Probably not worth spending a lot of time figuring this out, I'd just go with something. What you have here might even be fine.

**Pull request merged**